### ISSUE TITLE
1.3.1 version not available

### ISSUE THREAD

**Issue by p3lim** [olav-st/screencloud#242]
I see the tag, but there is no download available.

**Comment by furai** [comment]
I haven't seen any activity from @olav-st in a while. No idea what's going on.

**Comment by olav-st** [comment]
I haven't pushed out a release yet for two reasons:

1. I simply don't have time at the moment. Pushing out a new release on all platforms takes a lot of manual work (ideally it shouldn't, but it does). I won't have time to do that before December (earliest) or January (realistically).
2. In order to reduce the amount of manual work I have to do when pushing out a new release, I'm looking into dropping support for `.deb` and `.rpm` and use [snap packages](http://snapcraft.io/) instead. However, there are a few [bugs](https://bugs.launchpad.net/snappy/+bug/1600136) in snapd that's holding me back.

**Comment by furai** [comment]
It's understandable that you don't want to have too much work on your hands. Also I get that you don't have time for ScreenCloud right now. Weird things happen in real life that stop us from doing stuff. I don't have anything against you and I totally support you. Do release when you'll have time for it. No demands on my end.

The only thing now is that on ubuntu 16.04.1 I haven't been using ScreenCloud for over 2 months now because I couldn't get it to work after the latest update. I've tried few solutions here in issues on github but none of them worked for me. Probably I've done something wrong or I don't know actually...

**Comment by olav-st** [comment]
> The only thing now is that on ubuntu 16.04.1 I haven't been using ScreenCloud for over 2 months now because I couldn't get it to work after the latest update.

This is is the reason why I've come to hate `.deb` packages with a passion. Almost every Ubuntu release manages to break compatibility in some way. It's a huge headache every time and I spend hours figuring out why packages ScreenCloud depended on suddenly have been split, merged or just removed entirely.

I could go on and on about this, but I'll leave the rest of the rant for another time. The good news is that snap packages should theoretically fix these issues and make maintaining the Linux version of ScreenCloud a lot easier in the future. :)

**Comment by furai** [comment]
I'm looking forward to new release fixing Linux issues. :)

**Comment by Angristan** [comment]
What about the Windows version ? :)

**Comment by alons45** [comment]
Waiting snap package :dancer:

**Comment by p3lim** [comment]
If you're a plugin developer and want to support `{rnd_s}` for your plugin until @olav-st can get 1.3.1 out for all platforms, use this:

```Python
import re
def fix_filename(name):
	# temp fix until 1.3.1 gets released
	pattern = re.compile("^b'(.*)'(\.%s)$" % ScreenCloud.getScreenshotFormat())
	match = pattern.match(name)
	return match and ''.join(match.groups()) or name
```
Then everywhere you would normally use `ScreenCloud.formatFilename(s)` you'd encase it like this: `fix_filename(ScreenCloud.formatFilename(s))`.
